lib/model/image_sscd.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+import io
+
+# from lib.model.generic_image import GenericImageModel
+from lib.model.model import Model
+from lib import schemas
+from torchvision import transforms
+import torch
+from lib.logger import logger
+import numpy as np
+from PIL import Image
+import urllib.request
+
+imagenet_normalization_mean = [0.485, 0.456, 0.406]
+imagenet_normalization_std = [0.229, 0.224, 0.225]
+class Model(Model):
+    def __init__(self):
+        super().__init__()
+        #FIXME: Load from a Meedan S3 bucket
```

Comment:
Is this comment valid? Do we need to do something else here or delete this line?

Reply:
@computermacgyver what do you think? there has been an unresolved conversation about putting the file public in the s3 bucket and the its rules. Should we reopen this conversation and see how to handle hosting the file on the s3 bucket and evantually refer to the file location here? or should we just keep it in the s3 bucket as a backup and we use it in case it was removed from its current url?

Reply:
Waiting for @computermacgyver to weigh in here!